plugins/custom_calyptia/calyptia.c

```diff
@@ -324,6 +326,29 @@ static int cb_calyptia_init(struct flb_custom_instance *ins,
         flb_output_set_property(ctx->o, "tls.verify", "false");
     }
 
+    if (ctx->fleet_id) {
+        flb_output_set_property(ctx->o, "fleet_id", ctx->fleet_id);
+
+        ctx->fleet =  flb_input_new(config, "calyptia_fleet", NULL, FLB_FALSE);
+        if (!ctx->fleet) {
+            flb_plg_error(ctx->ins, "could not load Calyptia Fleet plugin");
+            return -1;
+        }
+
+        flb_input_set_property(ctx->fleet, "api_key", ctx->api_key);
+        flb_input_set_property(ctx->fleet, "host", ctx->cloud_host);
+        flb_input_set_property(ctx->fleet, "port", ctx->cloud_port);
+        flb_input_set_property(ctx->fleet, "tls", 
+                               (ctx->cloud_tls == 1 ? "on" : "off"));
```

Comment:
Please replace this and any other inline conditionals.

Reply:
Would it work to replace them with this?

```c
static const char *bool_to_str(int val)
{
    if (val == 1) {
        return "on";
    }
    return "off";
}

----
        flb_input_set_property(ctx->fleet, "tls", bool_to_str(ctx->cloud_tls));
        flb_input_set_property(ctx->fleet, "tls.verify", 
                               bool_to_str(ctx->cloud_tls_verify));
----
```

Reply:
I decided to just replace with verbose code that has a single line for each possibility.